src/DynamoCoreWpf/ViewModels/Preview/CompactBubbleViewModel.cs

```diff
@@ -1,13 +1,24 @@
-﻿namespace Dynamo.ViewModels
+using System;
+using System.Globalization;
+using Dynamo.Configuration;
+using Dynamo.Wpf.Properties;
+
+namespace Dynamo.ViewModels
 {
     /// <summary>
     /// Class containing data to display in compact preview bubble
     /// </summary>
     public class CompactBubbleViewModel : ViewModelBase
     {
         #region Properties
-
+        internal const string doubleType = "double";
+        internal const string boolType = "bool";
+        internal const string intType = "int";
+        internal const string objectType = "object";
+        internal const string stringType = "string";
```

Comment:
Are these defined somewhere already so we can make them static to reuse?

Reply:
can't you use `nameof`?

Reply:
Yes, there constants were used in WatchViewModel. Removing it from WatchViewModel and CompactBubbleViewModel and using nameof property.

Reply:
Thanks! Looks good